tools/measure_coverage.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+# Copyright 2017-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ "$#" -gt "0" ] && ( [ "$1" == "-h" ] || [ "$1" == "--help" ] ); then
+    echo "Measure JavaScript and C coverage and create a html report"
+    echo "out of the results"
+    echo ""
+    echo "Usage: $0 [NODE_MODULES_DIR]"
+    echo ""
+    echo "Optional Arguments:"
+    echo "  NODE_MODULES_DIR    Specifies the node_module directory, where"
+    echo "                      the nodejs dependencies are installed."
+    echo ""
+    echo "The created html reports can be found in the 'coverage' directory,"
+    echo "which will be created in the IoT.js project source dir. The C and"
+    echo "JavaScript coverage reports are in the 'c' and 'js' subdirectories"
+    echo "respectively. The reports can be viewed by opening the 'index.html'"
+    echo "file in a web browser of your choice."
+    echo ""
+    echo "Running the script will require some additional dependencies:"
+    echo "    sudo apt install npm lcov nodejs gcc-multilib"
+    echo "    npm install --save-dev babel-cli nyc babel-plugin-istanbul"
```

Comment:
Why you save this dependency? I think it should be inside package.json, and you expect it to be already installed, or you install it without saving. You can perform npm install babel-cli in empty directory and it will work